tests/e2e/tests/pilot/testdata/networking/v1alpha3/rule-default-route-append-headers.yaml

```diff
@@ -11,25 +11,55 @@ spec:
           host: c
           subset: v1
         weight: 75
-        appendRequestHeaders:
+        appendRequestHeaders: #deprecated
           istio-custom-dest-req-header: user-defined-value
-        removeRequestHeaders:
+        removeRequestHeaders: #deprecated
         - istio-custom-dest-req-header-remove
-        appendResponseHeaders:
+        appendResponseHeaders: #deprecated
           istio-custom-dest-resp-header: user-defined-value
-        removeResponseHeaders:
+        removeResponseHeaders: #deprecated
         - istio-custom-dest-resp-header-remove
```

Comment:
remove these from tests..

Reply:
you mean to remove all deprecated tests?